templates/lightning/invoice.html

```diff
@@ -60,6 +60,9 @@ <h2>Create Invoice</h2>
     }
 
     function confirm(form_class, amount) {
+	if (!document.querySelector(form_class).reportValidity()) {
```

Comment:
fix spacing, (you are probably mixing spaces and tabs)

Reply:
changed tab to space(s)